Update guide to v14 code (#1146)
discordjs/guide: guide/creating-your-bot/command-handling.md

Create a new folder named `commands`, which is where you'll store all of your commands.

> We'll be using utility methods from the package to build the slash command data.

Reviewer: seems a bit vague as to which package is being talked about perhaps? although this line could also be removed altogether

```suggestion
We'll be using utility methods of the library to build the slash command data.
```
